bindings/src/main/java/io/pravega/storage/extendeds3/ExtendedS3StorageConfig.java

```diff
@@ -30,6 +30,7 @@
     public static final Property<String> BUCKET = Property.named("bucket", "");
     public static final Property<String> PREFIX = Property.named("prefix", "/");
     public static final Property<Boolean> USENONEMATCH = Property.named("useNoneMatch", false);
+    public static final Property<Integer> SMALL_OBJECT_THRESHOLD = Property.named("smallObjectThreshold", 1024 * 1024);
```

Comment:
Please add unit into the name as commented.

Reply:
value is in bytes so by convention without suffix.